build.gradle

```diff
@@ -454,3 +458,45 @@ configure(project.fineractJavaProjects) {
         }
     }
 }
+
+configure(project.fineractPublishProjects) {
+    apply plugin: 'maven'
+    apply plugin: 'maven-publish'
+    apply plugin: 'com.jfrog.bintray'
```

Comment:
You probably want to remove this? FYI Travis build failed due to `Plugin with id 'com.jfrog.bintray' not found.`

Reply:
Arrgh! ;-)

Reply:
The maven stuff can be left there (then we don't have to remove the whole block). And as said, it won't hurt anyway. If someone does accidentally "./gradlew publish" then it will just do nothing.